jormungandr/src/rest/v1/logic.rs

```diff
@@ -115,23 +125,17 @@ pub async fn get_fragment_logs(context: &Context) -> Result<Vec<FragmentLog>, Er
     .await
 }
 
-pub async fn get_account_votes(
+pub async fn get_account_votes_with_plan(
     context: &Context,
     vote_plan_id: VotePlanId,
```

Comment:
Yes, seems reasonable, updating it

Reply:
Not sure, that's a great idea, because these two functions should have different returning type.
So I propose not to remove these functions but still refactor it, and move out to the separate another function the general code.